bin/hdfcoinc/pycbc_multiifo_combine_statmap

```diff
@@ -14,6 +14,11 @@ parser.add_argument('--verbose', action='store_true')
 parser.add_argument('--statmap-files', nargs='+',
                     help="List of coinc files to be redistributed")
 parser.add_argument('--cluster-window', type=float)
+parser.add_argument('--censor-ifar-threshold', type=float,
+    help="If provided, set the foreground censor to be times around only"
```

Comment:
"only candidates more" --> "only those candidates which are more"

Reply:
rewrite .. "If provided, only window out foreground triggers with IFAR above the threshold"